test/functional/cmdLineTests/runtimemxbeanTests/src/GetPid.java

```diff
@@ -23,12 +23,20 @@
 import java.lang.management.RuntimeMXBean;
 import java.lang.management.ManagementFactory;
 
-class GetPid {
+class GetPid extends Thread {
 
-	public static void main(String[] args) {
-		RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
-		long id = mb.getPid();
-		System.out.println(id);
-	}
+        private static Object monitor = new Object();
+ 
+        public static void main(String args[]) {
+                synchronized (monitor) {
+                        try {
+                                RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
+                                long id = mb.getPid();
+                                System.out.println(id);
+                                monitor.wait(5000);
```

Comment:
I think using a `Thread.sleep(...)` is more clear than the more complicated monitor sync/wait and achieves the same goal.  
  
Also, add a comment saying why the delay is required